### PR TITLE
[BACKLOG-10514] Fix for allowing category and series to also be fed b…

### PR DESCRIPTION
**Pull request by dcleao**
…y non-discrete visual roles

@pentaho/millenniumfalcon please review.

### CODE REVIEW

**Comment by dcleao** [comment]
@pamval @davidmsantos90 @nantunes @graimundo @marcovala please review.

**Review by carlosrusso — APPROVED** [review]
Seems fine to me.

File: exts/detTooltip/detTooltip.css [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
 /* Tooltip styles */
 .ccc-ext-det-tooltip * {
-  font-family: "open sans" sans;
+  font-family: OpenSansRegular;
```
Comment (nantunes) [inline exts/detTooltip/detTooltip.css]: Can you keep the fallbacks? `OpenSansRegular "open sans" sans`.

**Comment by dcleao** [comment]
Updated commit to preserve the `sans` font family as a fallback.

**Pull request merged**